src/eventer/eventer_aco_opset.c

```diff
@@ -57,9 +57,16 @@ typedef struct aco_opset_info_t {
 } aco_opset_info_t;
 
 #define WORRISOME_STACK 2048
+static void
```

Comment:
Why are we wrapping `aco_yield()` in a new function here ?

Reply:
Makes it easier to debug.